meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -248,6 +248,13 @@ IMAGE_CMD_resin-sdcard () {
     # Finally add the fingerprint to vfat partition
     mcopy -i ${WORKDIR}/boot.img -v ${WORKDIR}/${RESIN_BOOT_FS_LABEL}.${FINGERPRINT_EXT} ::
 
+    # Generate and inject config.json
+    echo '{}' > ${WORKDIR}/config.json
+    if ${@bb.utils.contains('DISTRO_FEATURES','debug-image','true','false',d)}; then
+        echo $(cat ${WORKDIR}/config.json | jq ".hostname=\"resin\"") > ${WORKDIR}/config.json
```

Comment:
Would the image maker not overwrite this config.json that we generate at build time?